common/include/pcl/common/io.h

```diff
@@ -288,8 +288,8 @@ namespace pcl
 
   /** \brief Concatenate two pcl::PCLPointCloud2
     *
-    * \warn This function subtly differs from the deprecated `concatenatePointloud`
-    * The difference is thatthis function will concatenate IFF the non-skip fields
+    * \warning This function subtly differs from the deprecated ::concatenatePointCloud
```

Comment:
This is confusing to a cpp reader: does `::concatenatePointCloud` refers to a function in the global namespace (as cpp syntax says) or in the nearest namespace (as I gathered from context)

It may make sense to instead use the `\ref` or one of the other syntax for [autolink](http://www.doxygen.nl/manual/autolink.html), such as parenthesis or `#` to reduce confusion.

Reply:
This is for autolink. Changed to braces, looks much better, you're right.

Reply:
Just a reminder: You might have forgotten to push the change.